app/components/StackedProfiles.tsx

```diff
@@ -0,0 +1,58 @@
+import { tw, tx } from '@helpwave/common/twind/index'
+import type { Languages } from '@helpwave/common/hooks/useLanguage'
+import type { PropsWithLanguage } from '@helpwave/common/hooks/useTranslation'
+import { useTranslation } from '@helpwave/common/hooks/useTranslation'
+import { ProfilePicture } from './ProfilePicture'
+
+type StackedProfilesTranslation = {
+  others: (isMultipleUsers: boolean) => string
+}
+
+const defaultStackedProfilesTranslations: Record<Languages, StackedProfilesTranslation> = {
+  en: {
+    others: (isMultipleUsers: boolean) => isMultipleUsers ? 'other' : 'others'
+  },
+  de: {
+    others: (isMultipleUsers: boolean) => isMultipleUsers ? 'weiterer' : 'weitere'
+  }
```

Comment:
this logic shouldn't be part of the translation object: `otherSingular`, `otherPlural` is fine as well